learn/advanced/updating.md

```diff
@@ -106,47 +109,29 @@ docker run -it --rm \
 If you are updating to v0.28, keys imported from the old version will have their `key` and `uid` fields regenerated.
 :::
 
-### Proceed according to your database version
-
-Now that you know which Meilisearch version your database is compatible with, proceed accordingly:
-
-- **If your database version is v0.15.0 or above**, continue to the [next section](#updating-from-v0-15-0-or-above).
-- **If your version is v0.14.0 or below**, [go here](#updating-from-v0-14-0-or-below).
-
-## Updating from v0.15.0 or above
-
-Because Meilisearch v0.15.0 and above include the [dumps feature](/learn/advanced/dumps.md), updating is relatively simple.
-
-In this guide, we will:
-
-1. [Set all fields as displayed attributes](#step-1-set-all-fields-as-displayed-attributes)
-2. [Create a dump using the Meilisearch version that's compatible with your database](#step-2-create-the-dump)
-3. [Delete the database folder](#step-3-delete-the-database-folder)
-4. [Import the dump using the most recent Meilisearch version](#step-4-import-the-dump)
-
-### Step 1: Set all fields as displayed attributes
+## Step 2: Set all fields as displayed attributes
 
 ::: note
-If your dump was created in Meilisearch v0.21 or above, continue to step 2.
+If your dump was created in Meilisearch v0.21 or above, continue to [step 3](#step-3-create-the-dump).
 :::
 
 When creating dumps, Meilisearch calls the same method as the [get documents endpoint](/reference/api/documents.md#get-documents). This means that all fields must be [displayed](/learn/configuration/displayed_searchable_attributes.md#displayed-fields) in order to be saved in the dump.
 
-Start by using the [get displayed attributes endpoint](/reference/api/displayed_attributes.md#get-displayed-attributes) to verify that **all attributes are displayed**.
+Start by using the get displayed attributes endpoint to verify that all attributes are displayed:
```

Comment:
If the link is not necessary, I think we can make this sentence sound a bit more natural:

```suggestion
Start by verifying that all attributes are included in the displayed attributes list:
```

Or something to that effect. I find the `the [something] endpoint` construct a bit wooden when we're not using it to link to the reference.